src/services/config.service.ts

```diff
@@ -28,6 +47,16 @@ const featuresSchema: SchemaOf<Features> = object({
   enableSharing: boolean().notRequired(),
   recommendationsPlaylist: string().nullable(),
   searchPlaylist: string().nullable(),
+  continue_watching_list: string().test('has-continue_watching-list-element', 'errorMessage', (value, context) => {
```

Comment:
imho, this is more validation than we want a schema check to do. Instead of just checking formatting, it's starting to enforce business logic.

Reply:
But what does it hurt if they set a playlist id but then don't use it in the content items? It may be a mistake but it also may be a use case we haven't planned for yet (like they want to put favorites on a new page or in a library for example.) I understand we need to enforce that the playlist id is set if the content item is used, but don't think the opposite is necessarily a hard fail. Secondly, even if we decide it is, I'd prefer to do this validation in the controller or service and let the scheme validation just be formatting checks.

Reply:
I agree that nothing prevents us from using just playlistIds in features with no corresponding content items. So I moved it to `loadConfig` function and added a log message instead of error. Please have a look.